frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card-item.tsx

```diff
@@ -12,16 +13,16 @@ const formatCount = (count: number) => {
   return pluralizeWithNotation(count / 1000000, 'M', 'Object');
 };
 
-const BucketsRowStatus: React.FC<BucketsRowStatusProps> = React.memo(({ status }) => (
+const BucketsRowStatus: React.FC<BucketsRowStatusProps> = React.memo(({ status, link }) => (
   <div className="nb-buckets-card__row-status-item">
     {_.isNil(status) ? (
       <span className="nb-buckets-card__row-subtitle">Unavailable</span>
     ) : Number(status) > 0 ? (
       <React.Fragment>
-        <div>
-          <ArrowCircleDownIcon />
-        </div>
-        <div className="nb-buckets-card__row-status-item-text">{status}</div>
+        <Link to={link} style={{ textDecoration: 'none' }}>
```

Comment:
please extract the inline style to `scss` file

Reply:
Could not achieve this on moving into `scss` file, I guess this is getting override by some css.

Reply:
@rawagner ^^

Reply:
ok, i see we are using it elsewhere too.